src/sidebar/services/stream-filter.js

```diff
@@ -7,16 +50,16 @@
  */
 export default class StreamFilter {
   constructor() {
-    this.resetFilter();
+    this.filter = defaultFilter();
```

Comment:
This change makes it more obvious to the reader and to TS that `this.filter` always gets initialised before it is used.